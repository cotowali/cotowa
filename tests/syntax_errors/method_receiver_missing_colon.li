type Num = int
fn (n Num) f() {}
