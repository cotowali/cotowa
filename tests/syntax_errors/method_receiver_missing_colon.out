error: method_receiver_missing_colon.li 2,7: unexpected token `Num`, expecting `colon`
    2 | fn (n Num) f() {}
      |       ^^^
